MdeModulePkg/Include/Library/DeviceStateLib.h

```diff
@@ -43,6 +45,14 @@ EFIAPI
 GetDeviceState (
   );
 
+/**
+Function to get InsecureDeviceState PCD
+**/
+INSECURE_DEVICE_STATE
+EFIAPI
+GetInsecureDeviceState (

```

Comment:
Maybe GetInsecureDeviceStateSettings or something that conveys what this returns a bit better.